Modules/ThirdParty/HDF5/src/itkhdf5/config/cmake_ext_mod/ConfigureChecks.cmake

```diff
@@ -275,7 +275,7 @@ if (MINGW OR NOT WINDOWS)
   else ()
   set(HDF_ENABLE_LARGE_FILE ON)
   endif ()
-  if (HDF_ENABLE_LARGE_FILE AND NOT DEFINED TEST_LFS_WORKS_RUN)
+  if (HDF_ENABLE_LARGE_FILE)
```

Comment:
Submitting this patch to [HDF5](https://github.com/HDFGroup/hdf5) is desirable.

Reply:
I created a new issue https://github.com/HDFGroup/hdf5/issues/2395 in HDF5.